angular/projects/admin/src/app/services/app-auth.service.ts

```diff
@@ -24,4 +24,21 @@ export class AppAuthService {
   logOut(): Promise<void> {
     return this.fireAuth.auth.signOut();
   }
+
+  async reloadUserService() {
```

Comment:
The function should just be called `reloadUser` the class itself is the "service"

Reply:
Resolve on this commit https://github.com/oddbit/tanam/pull/195/commits/87684dd3dcaa2d579fc888e3fc60e9e159a3dc2e